operators/cmd/crownlabs-delete-labinstance/test_instance_is_expired.py

```diff
@@ -0,0 +1,30 @@
+import unittest
+import deleteCron as delCron
+from datetime import datetime
+from datetime import timedelta
+
+
```

Comment:
IMHO, The purpose of the test is a bit hard to get. Does TestOld test if the deletion is correctly performed? I also propose to better highlight the definition of the two test-cases.